waspc/test/Generator/MockWriteableMonad.hs

```diff
@@ -118,6 +128,10 @@ createDirectoryIfMissing_addCall createParents path logs =
         (createParents, path) : createDirectoryIfMissing_calls logs
     }
 
+copyDirectoryRecursive_addCall :: Path' Abs Dir' -> Path' Abs Dir' -> MockWriteableMonadLogs -> MockWriteableMonadLogs
+copyDirectoryRecursive_addCall srcPath dstPath logs =
+  logs {copyDirectoryRecursive_calls = (srcPath, dstPath) : copyDirectoryRecursive_calls logs}
+
```

Comment:
Just a comment on this file -> looking at it, most of this logic should really be somehow automatically generated. And allegedly there are some libraries out there that use TH for this, so we might want to look it up at some point.